src/svc_xprt.c

```diff
@@ -102,14 +103,16 @@ void svc_xprt_init()
 
  unlock:
 	mutex_unlock(&svc_xprt_fd.lock);
+	return (code);
 }
 
-#define cond_init_svc_xprt() { \
-		do { \
-			if (!initialized) \
-				svc_xprt_init(); \
-		} while (0); \
-	}
+static inline bool
+svc_xprt_init_failure(void)
+{
+	if (initialized)
+		return (false);
```

Comment:
Yes, it should be false here. For ease of programming and speed of execution in the fast path, is only true on failure. Thus the name of the function: xxx_init_failure().